### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by the-furry-hubofeverything** [junglemc/JungleTree#1]
Are there general goals for this project? I'd like to help but I don't know where to start.

**Comment by EllaFennec** [comment]
Not really. It's a very personal project to me, and I'd like to work on it solo for the time being. I'll take contributions when I'm ready, though.

**Comment by EllaFennec** [comment]
I've began to get the project ready for contributions today, kinda by accident. Keep an eye on this issue thread, as I'll update it when I have more info :smiley:
